vsintegration/src/FSharp.Editor/Formatting/EditorFormattingService.fs

```diff
@@ -22,6 +23,10 @@ type internal FSharpEditorFormattingService
         checkerProvider: FSharpCheckerProvider,
         projectInfoManager: FSharpProjectOptionsManager
     ) =
+    
+    static let toIList (xs : 'a seq) : IList<'a> = ResizeArray(xs) :> _
+    
+    static let getIndentation : string -> int = Seq.takeWhile ((=) ' ') >> Seq.length
```

Comment:
And this is just after Don's talk where he said "do not use point free, just don't" ;)